### PR TITLE
Qualia4 2-improved

### PR DESCRIPTION
**Pull request by qualia4**
improved sum with functions in python, c++ and js files

### CODE REVIEW

**Comment by ykhadnevich** [comment]
Very good improvements my friend!

**Comment by YaroslavKSE** [comment]
This code works! Nice!

File: 2-improved.cpp [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+
+int sum(num1, num2)
```
Comment (vkukh) [inline 2-improved.cpp:3]: Each parameter should consist of a type followed by an identifier `int sum(int num1, int num2)`

File: 2-improved.cpp [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+{
+  return num1 + num2;
+}
+
+int main() 
+{
+  int first_num = 2;
+  int second_num = 3;
+  cout << sum(first_num, second_num) << endl;
```
Comment (vkukh) [inline 2-improved.cpp:12]: If you used `stdio.h`, then should use `printf()` instead of `cout`

**Pull request merged**